src/main/resources/templates/error/error.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
Generic error template should be one level up. Only specific error are placed under /error if I am not mistaken.

Reply:
yea you're correct let me fix that